assets/js/modules/analytics-4/components/audience-segmentation/dashboard/AudienceTilesWidget/AudienceTile/index.stories.js

```diff
@@ -303,6 +303,22 @@ TopContentPartialData.scenario = {
 	label: 'Modules/Analytics4/Components/AudienceSegmentation/Dashboard/AudienceTile/TopContentPartialData',
 };
 
+export const viewOnlyNoDimensions = Template.bind( {} );
+viewOnlyNoDimensions.storyName =
+	"Top content hidden when dimension doesn't exist for view only user";
+viewOnlyNoDimensions.args = {
+	...readyProps,
+	viewContext: VIEW_CONTEXT_MAIN_DASHBOARD_VIEW_ONLY,
+	setupRegistry: ( registry ) => {
+		registry.dispatch( MODULES_ANALYTICS_4 ).setSettings( {
+			availableAudiences: null,
+			availableCustomDimensions: [],
+			propertyID: '123456789',
+		} );
+	},
+};
+viewOnlyNoDimensions.scenario = {};
```

Comment:
This story const should have a capital initial. Again, I'll apply this to save the round trip.

```suggestion
export const ViewOnlyNoDimensions = Template.bind( {} );
ViewOnlyNoDimensions.storyName =
	"Top content hidden when dimension doesn't exist for view only user";
ViewOnlyNoDimensions.args = {
	...readyProps,
	viewContext: VIEW_CONTEXT_MAIN_DASHBOARD_VIEW_ONLY,
	setupRegistry: ( registry ) => {
		registry.dispatch( MODULES_ANALYTICS_4 ).setSettings( {
			availableAudiences: null,
			availableCustomDimensions: [],
			propertyID: '123456789',
		} );
	},
};
ViewOnlyNoDimensions.scenario = {};
```